<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groundstationSettingsBase</class>
 <widget class="QDialog" name="groundstationSettingsBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Groundstation Data</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="TextLabel1">
     <property name="text">
      <string>Longitude (East):</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="TextLabel2">
     <property name="text">
      <string>Latitude (North):</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="TextLabel3">
     <property name="text">
      <string>Height:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="TextLabel4">
     <property name="text">
      <string>Callsign:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="longitudeEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="latitudeEdit"/>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLineEdit" name="heightEdit"/>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QLineEdit" name="callsignEdit"/>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="Line" name="Line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="buttonOk">
     <property name="text">
      <string>&amp;Ok</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
